src/rest/gmm_http_server.erl

```diff
@@ -55,22 +55,21 @@ start_server() ->
             %%% @todo rest_load_simulator
 
             %  POST {BODY->request :: LoadSimulationRequestDto} -> void
-            {"/simulate_load", not_implemented, #{}},
+            {"/simulate_load", rest_load_simulator, #{}},
 
-            %%% @todo rest_meta_info - partially implemented
+            %%% @todo rest_meta_info - rest api ready, logic not implemented
```

Comment:
```suggestion
            %%% @todo rest_meta_info - implement logic
```

Reply:
Jak wyżej, mam już na to pomysł, się zrobi